tests/components/application-connector/test/compass-runtime-agent/synchronisation_test.go

```diff
@@ -49,6 +58,32 @@ func (cs *CompassRuntimeAgentSuite) TestApplication() {
 	})
 }
 
+func (cs *CompassRuntimeAgentSuite) updateAndWaitForCompare(appReader ApplicationReader, compassAppName, applicationID, updatedName string) error {
+	t := cs.T()
+	t.Helper()
+
+	exec := func() error {
+		_, err := cs.directorClient.UpdateApplication(applicationID, "Updated")
+		return err
+	}
+
+	verify := func() bool {
+		app, err := appReader.Get(context.Background(), compassAppName, v1.GetOptions{})
+		if err != nil {
+			t.Logf("Couldn't get updated: %v", err)
+		}
+
+		return err == nil && app.Spec.Description == "Updated"
```

Comment:
I'd use some constant value like `UPDATED_APP_DESCRIPTION` or something like this just to show explicitly that it's a hardcoded test value. What do you think?

Reply:
Do we really need to check the description here though? Isn't it a job for the App Comparator and the `Compare` method executed later?

Reply:
We do have to somehow wait for sync. App Comparator fails the tests if the app doesn't match so it cannot be used for waiting.